string[]              joint_names
float64[]             accelerations
---

